INSTALL.txt

```diff
@@ -1,4 +1,60 @@
-To install this package, type the following in the command line:
+# Dependencies
 
-  python setup.py install
+## Python 3
 
+Learn more: https://docs.python.org/3
+
+MacOS X: Install with [homebrew](http://brew.sh),
+
+    sudo xcodebuild -license ;
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ;
+    brew install python3 ;
+
+## PyQt5
+
+Learn more: https://www.riverbankcomputing.com/software/pyqt
+
+MacOS X: Install with homebrew, or re-install with DBus support,
+
+    brew remove qt5 && brew install qt5 --with-d-bus ;
```

Comment:
On Oct 19, 2015 6:30 PM, "Adrien Tétar" <notifications@github.com> wrote:
>
> In INSTALL.txt:
>
> >
> > +Learn more: https://docs.python.org/3
> > +
> > +MacOS X: Install with [homebrew](http://brew.sh),
> > +
> > +    sudo xcodebuild -license ;
> > +    ruby -e "$(curl -fsSL
https://raw.githubusercontent.com/Homebrew/install/master/install)" ;
> > +    brew install python3 ;
> > +
> > +## PyQt5
> > +
> > +Learn more: https://www.riverbankcomputing.com/software/pyqt
> > +
> > +MacOS X: Install with homebrew, or re-install with DBus support,
> > +
> > +    brew remove qt5 && brew install qt5 --with-d-bus ;
>
> brew install pyqt5 will install the dependency automatically.

It did not 12 hours ago but there was a PR to brew to fix it